Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -61,6 +63,8 @@ public partial class RangeSelector : Control
         /// </summary>
         public static readonly DependencyProperty StepFrequencyProperty = DependencyProperty.Register(nameof(StepFrequency), typeof(double), typeof(RangeSelector), new PropertyMetadata(DefaultStepFrequency));
 
+        private readonly DispatcherTimer dispatcherTimer = new DispatcherTimer();
```

Comment:
Could not find a better name, think its fine?

Reply:
We could give it a `keyDebounceTimer` name to be more specific about where it's used. Especially as we can only use Debounce for the one instance of the timer.